pkg/daemon/containerrecreate/crr_daemon_util.go

```diff
@@ -69,6 +69,8 @@ func getCurrentCRRContainersRecreateStates(
 	}
 
 	syncContainerStatuses := getCRRSyncContainerStatuses(crr)
+	// the statuses of force killed containers for this crr resource
+	getForceKillContainerStatuses := getCRRForceKillContainersStatuses(crr)
```

Comment:
getForceKillContainerStatuses -> forceKillContainerStatuses

Reply:
// the statuses of force killed containers for this crr resource
forceKillContainerStatuses := getCRRForceKillContainersStatuses(crr)
...
done